Ruddarr/Models/Instances/RadarrInstance.swift

```diff
@@ -3,6 +3,7 @@ import SwiftUI
 import Foundation
 
 @Observable
+@MainActor
```

Comment:
this was a cascading effect of marking Movies as main actor, but it feels natural here also.

Reply:
I've been seeing zero crashes in the last several testflight builds due to some other `@MainActor` changes we've done. I'm gonna put this on hold and see if any more crashes come in first.